CONTRIBUTING.md

```diff
@@ -82,11 +82,22 @@ Follow these steps to set up your local copy of the repository:
 
 1. Install [Ruby](https://www.ruby-lang.org/en/) if you don't already have it. We recommend [RVM](https://rvm.io/), but you can use any method you prefer:
 
+   ###### For non-Apple silicon machines:
+
    ```
    curl -sSL https://get.rvm.io | bash -s stable
    rvm install 3.2.4
    ruby -v
    ```
+   ###### For Apple silicon machines: 
+   Source: [Installing Ruby on Apple Silicon by Emmanuel Bernard ](https://emmanuelbernard.com/blog/2023/12/01/installing-ruby-on-apple-silicon/) 
+
+   ```
+   # Assumes Brew is installed
+   curl -sSL https://get.rvm.io | bash -s stable
+   rvm install 3.2.4 --with-openssl-dir=$(brew --prefix openssl@1.1)
```

Comment:
Can you indicate other versions of SSL besides 1.1? 3.3 is the current version, https://github.com/ruby/openssl/blob/master/README.md

Reply:
Okay what about:

   ```
   Please check with the current [SSL versions compatible with ruby](https://github.com/ruby/openssl/blob/master/README.md) 3.2.4 if you don't have `openssl@3`

   # Assumes Brew is installed
   curl -sSL https://get.rvm.io | bash -s stable
   rvm install 3.2.4 --with-openssl-dir=$(brew --prefix openssl@3)
```

Reply:
We could always make the instruction version agnostic, for example:

```
# Assumes Brew is installed
curl -sSL https://get.rvm.io | bash -s stable
rvm install 3.2.4 --with-openssl-dir=$(brew --prefix openssl@<openssl-version>)
```

Reply:
Hey @Naarcha-AWS thank you for your persistence on this issue, I pushed a commit that addresses this concern

[f913275](https://github.com/opensearch-project/documentation-website/pull/8618/commits/f913275ed54145711ebf6582ee191ddaf6bc6f67)